dcos/version.go

```diff
@@ -0,0 +1,7 @@
+package dcos
+
+// Version represents the client library version
+const Version = "0.0.1"
+
+// ClientName represents the libraries name
+const ClientName = "dcos/client-go"
```

Comment:
I actually had to laugh out loud when I saw this 😄 So much for "we call it `dcos-client-go` everywhere, only the repository is named `dcos/client-go`" 😉

Reply:
haha... indeed and I wasn't sure anymore how we end up as we decided to stay with the repo as is... OK so the client is named dcos-client-go will change it so we are sure once and for all ;)